blocks/browse/da-browse/da-browse.js

```diff
@@ -197,8 +197,9 @@ export default class DaBrowse extends LitElement {
       formData.append('destination', item.destination);
       const opts = { method: 'POST', body: formData };
       await daFetch(`${DA_ORIGIN}/copy${item.path}`, opts);
-      item.isChecked = false;
-      this._listItems.unshift(item);
+
+      const pastedItem = { ...item, path: item.destination, isChecked: false };
```

Comment:
I think we still want `item.isChecked = false;`. Otherwise, 👍

Reply:
Ah, missed that. Thanks!